### DISCUSSION TITLE
FAQ

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [LinuxSuRen/api-testing#144] [Q&A]
## 通过容器启动时，如何指定 Web 端口
使用示例：
```shell
docker run --pull always --network host ghcr.io/linuxsuren/api-testing:master atest server --http-port 7890 \
  --console-path /var/www/html
```

## 有没有在线 Demo
您可以访问下面的地址在线体验：
* https://api-testing.zeabur.app/

**Comment by hahahashen** [comment]
我在本地运行时创建了一个测试用例，但是没有找到指定请求方式get、post的地方，也没有看到有请求发送按钮
![image](https://github.com/LinuxSuRen/api-testing/assets/59725322/d9082e38-ebc7-4a85-a6f4-e9aaf47319c5)

**Reply by hahahashen** [reply]
不知道我的使用方式哪里出了问题？

**Reply by LinuxSuRen** [reply]
你新建的是一个测试用例集，下面有个创建测试用例的按钮。

**Comment by Agility6** [comment]
我在新建测试用例遇到以下问题，我应该如何在web中生成代码
<img width="772" alt="image" src="https://github.com/LinuxSuRen/api-testing/assets/75616395/66d7a781-2d7e-4fef-aabb-3c8cf61bb656">

**Reply by LinuxSuRen** [reply]
可能是前端判断有问题，浏览器 console 有报错信息吗？下面是相关代码

https://github.com/LinuxSuRen/api-testing/blob/c8378aee85a41ea6be10adc7189b9405c88a1735/console/atest-ui/src/views/TestCase.vue#L212

> 考虑到本次（2024）OSPP 的议题中包含了前后端部分，也建议熟悉下 UI 部分，甚至做下简单的优化。

**Reply by Agility6** [reply]
似乎是我启动有问题，我注意到log显示500错误
<img width="1240" alt="image" src="https://github.com/LinuxSuRen/api-testing/assets/75616395/14c63647-546d-46e1-b380-a67a0c8010f2">

我是这样启动服务
1. `go build`
2. `./api-testing server`

**Reply by LinuxSuRen** [reply]
你试试：`./api-testing server --local-storage 'bin/*.yaml'`

**Reply by Agility6** [reply]
ok，没问题了
